packages/nimble-components/src/table/models/table-validator.ts

```diff
@@ -10,14 +10,16 @@ export class TableValidator<TData extends TableRecord> {
     private invalidRecordId = false;
     private duplicateColumnId = false;
     private missingColumnId = false;
+    private duplicateSortIndex = false;
 
     public getValidity(): TableValidity {
         return {
             duplicateRecordId: this.duplicateRecordId,
             missingRecordId: this.missingRecordId,
             invalidRecordId: this.invalidRecordId,
             duplicateColumnId: this.duplicateColumnId,
-            missingColumnId: this.missingColumnId
+            missingColumnId: this.missingColumnId,
+            duplicateSortIndex: this.duplicateSortIndex
```

Comment:
Reminder to add this to the storybook where we document all these invalid states.

Alternatively, I'd also be open to stopping that practice and instead changing those docs to point at this source file for a list of invalid states.

Reply:
I've added the docs for now. I'm not sure what the best long-term answer is. Pointing to the source file for the list of invalid states wouldn't include what causes the table to enter a given invalid state (at least right now it doesn't....maybe it should though).